src/apps/loyalty/analytics.ts

```diff
@@ -0,0 +1,79 @@
+import { extend, pick } from "lodash"
+import * as sharify from "sharify"
+
+import { ResponseLocalData } from "./types"
+const data = sharify.data as ResponseLocalData
+
+const Analytics = require("analytics-node")
```

Comment:
One more thing, it took some finagling with `require` vs `import`, etc. in order to be able to initialize this properly and do `analytics.identify` and stuff. However along that route, its now typed as `any` and we lose all that info, even though the typings are included.

It's possible I'm just doing something wrong with this setup...but I'm not sure. Either way, this 'works' for now.